src/com/amazon/ion/benchmark/Gradient.java

```diff
@@ -0,0 +1,8 @@
+package com.amazon.ion.benchmark;
+
+public enum Gradient {
```

Comment:
suggestion— "Gradient" is not a self-explanatory name. Can you add some javadoc here?

Reply:
Sure, but I'm also not set on this name at all. I just want a name for the words that will be used to describe a range from none to all (and maybe values in-between in the future). Any ideas?

Reply:
So, if you really want it to just represent a number from `0.0` to `1.0`, the "gradient" or just about any synonym is fine. However, if we want to be able to e.g. test with other strategies, such as "intern field names and annotations, but inline symbol values" or "only use length prefixes for top-level containers", then I don't think that "gradient" would make sense.

Reply:
I decided to keep "Gradient" now, for lack of something better. I added a short description. If we end up needing more specific options related to inline symbols or delimited containers, we'll likely need to swap this out for a more targeted enum anyway.